modules/filtering-insights-advisor-recommendations.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/using-insights-to-identify-issues-with-your-cluster.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="filtering-unnecessary-advisor-recommendations_{context}"]
+= Filtering Insights advisor recommendations
+
+As an OpenShift cluster manager, you can filter the recommendations that are displayed on the recommendations list. By applying filters, you can reduce the number of reported recommendations and concentrate on your highest priority recommendations.
+
+The following procedure demonstrates how to set and remove *Category* filters; however, the procedure is applicable to any of the primary and secondary filters. 
+
+.Prerequisites
+You are logged in to the https://console.redhat.com/openshift[OpenShift Cluster Manager Hybrid Cloud Console].
+
+.Procedure
+. Navigate to the link:https://console.redhat.com/openshift/insights/advisor/recommendations[*Red Hat Hybrid Cloud Console* -> *OpenShift* -> *Advisor recommendations*] page.
+. Using the filter input fields above the list of recommendations, select the *Category* filter type.
+. Expand the right dropdown and check the box next to each category of recommendation you want to view. Leave unnecessary categories unchecked.
```

Comment:
```suggestion
. From the [add label name!] drop-down list, select each category of recommendation you want to view.
```

- [Per the ISG](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#location-of-interface-elements), "When you refer to the location of an element in a graphical user interface, do not use sensory words, such as _top_, _bottom_, _right_, or _left_." 
- Use [the ISG standards for drop-down lists](https://www.ibm.com/docs/en/ibm-style?topic=elements-ui#drop_down_list) whenever you refer to these UI elements.
- Per the ISG, do not use "check" to refer to marking a checkbox. Use "select".
- Also if it is a checkbox, refer to it as a "checkbox" and not a "box".

Reply:
Done. Just as an fyi, the dropdowns on the Hybrid Cloud Console filters don't have label names (or hover-over names); there are two of them, and they're side by side. We call them the primary and secondary filters in another Console project, but neither the engineers nor peer review like that phrasing so I'm trying to be specific for the users, given both the limitations and rapidly changing nature of the GUI.